packages/patternfly-4/react-core/src/layouts/Page/Page.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/layouts/Page/page.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { BarsIcon } from '@patternfly/react-icons';
+import { Button, ButtonVariant } from '@patternfly/react-core';
+
+export const PageLayouts = {
+  vertical: 'vertical',
+  horizontal: 'horizontal'
+};
+
+const propTypes = {
+  /** Content rendered inside the Page layout */
+  children: PropTypes.node,
+  /** Additional classes added to the Page layout */
+  className: PropTypes.string,
+  /** Page layout type */
+  layout: PropTypes.oneOf(Object.values(PageLayouts)),
+  /** Component to render the background image */
+  background: PropTypes.node,
+  /** Component to render the logo/brand */
+  logo: PropTypes.node,
+  /** Component to render the navigation */
+  nav: PropTypes.node,
+  /** Vertical layout only: True if the nav is expanded  */
+  isNavOpen: PropTypes.bool,
+  /** Vertical page layout only: Callback function to handle the navigation toggle button */
+  onNavToggle: PropTypes.func,
+  /** Component to render the toolbar */
+  toolbar: PropTypes.node,
+  /** Component to render the avatar */
+  avatar: PropTypes.node
+};
+
+const defaultProps = {
+  children: null,
+  className: '',
+  layout: 'vertical',
+  background: null,
+  logo: 'Logo',
+  nav: 'Nav',
+  onNavToggle: () => undefined,
+  isNavOpen: true,
+  toolbar: 'Toolbar',
+  avatar: null
+};
+
+const Page = ({
+  className,
+  children,
+  layout,
+  background,
+  logo,
+  nav,
+  isNavOpen,
+  onNavToggle,
+  toolbar,
+  avatar,
+  ...props
+}) => (
+    <React.Fragment>
+      {background}
+      <div {...props} className={css(styles.page, className)}>
+        {/* Added style={{ overflowX: 'unset' }} as a workaround to make dropdowns work */}
```

Comment:
When this issue in pf-next is fixed i can remove it
https://github.com/patternfly/patternfly-next/issues/780

Reply:
I wonder if you should add something to the comment so it is clear that a fix is coming and the work around is temporary.

Reply:
I wonder if we want to add something to the code comment to make it clear that this is a temporary work around.